### PR TITLE
Add real tooltips to pagetree

### PR DESCRIPTION
**Pull request by vxsx**
also add minor ui fixes
- dropdown header change
- wizard cms class
- minor restructuring
- bigger timeouts on integration tests

screenshots:

screen shot 2016-04-26 at 10 49 32
screen shot 2016-04-26 at 10 49 46
screen shot 2016-04-26 at 10 49 59
screen shot 2016-04-26 at 10 50 30

### CODE REVIEW

File: cms/tests/frontend/integration/pagetree.js [hunk 1]
```diff
@@ -1314,7 +1314,7 @@ casper.test.begin('Pagetree remembers which nodes are opened and which ones are
                     window.location.reload();
                 })
                 .waitUntilVisible('.cms-pagetree-jstree')
-                .wait(1000, function () {
+                .wait(2000, function () {
```
Comment (FinalAngel) [inline cms/tests/frontend/integration/pagetree.js]: can we add there global variable or so?

**Comment by FinalAngel** [comment]
LGTM

File: cms/tests/frontend/integration/pagetree.js (hunk 1 above)
Comment (vxsx) [inline cms/tests/frontend/integration/pagetree.js]: not really :) they are very arbitrary and subject to change, i wouldn't touch it

Comment (FinalAngel) [inline cms/tests/frontend/integration/pagetree.js]: alright 👍

**Pull request merged**